src/current/cockroachcloud/managing-access.md

```diff
@@ -107,7 +111,11 @@ Service accounts, like users, are given only the **Org Member** role by default
 
 1. On the **Access Management** page, select the **Service Accounts** tab.
 1. In the row for the target service account, click, click the three-dots **Action** button and select **Edit Roles**.
-1. Note that a number of fine-grained roles can be assigned to a given service account. These are the same [roles that can be assigned to users](authorization.html#organization-user-roles). Each role is represented by a row. Each row has a **scope**, which is either **Organization** or the name of a particular cluster. If the role is Cluster Administrator or Cluster Developer, giving it the scope of organization means that it applies to all clusters in the organization.
+1. A number of fine-grained roles can be assigned to a given service account. These are the same [roles that can be assigned to users](authorization.html#organization-user-roles). Each role is represented by a row. Each row has a **scope**, which is either **Organization** or the name of a particular cluster. If the role is Cluster Administrator, Cluster Operator, or Cluster Developer, assigning it at the organization scope means that it applies to all clusters in the organization.
+
+    {{site.data.alerts.callout_info}}
+    The fields for a group's inherited roles are read-only, because inherited roles cannot be edited directly. Instead, you must either remove the role from the parent group from which it is inherited, or remove the member from the parent group.
+    {{site.data.alerts.end}}
 
```

Comment:
```suggestion
    The fields for a group's inherited roles are read-only, because inherited roles cannot be edited directly. Instead, you must either remove the role from the parent group from which it is inherited, or remove the member from the parent group.
```